lib/controllers/poolplayers.js

```diff
@@ -0,0 +1,36 @@
+const { Router } = require('express');
+const { findAll } = require('../models/Poolplayer');
+const Poolplayer = require('../models/Poolplayer');
+
+module.exports = Router() 
+  
+  .post('/', async(req, res) => {
+    const poolplayer = await Poolplayer.insert(req.body);
+    res.send(poolplayer);
+  }) 
+  
+  .get('/', async (req, res) => {
+    const poolplayers = await findAll();
```

Comment:
I am a little surprised that this is not breaking on line 13.  You should `await Poolplayer.findAll()`.